makefile

```diff
@@ -63,7 +64,7 @@ lint:
 	@echo -e "$(ATTN_COLOR)==> $@ $(NO_COLOR)"
 	@${EXT_BIN_DIR}/golangci-lint config path
 	@${EXT_BIN_DIR}/golangci-lint config verify
-	@go list -f '{{.Dir}}/...' -m | xargs ${EXT_BIN_DIR}/golangci-lint run --config ${PWD}/.golangci.yaml
+	@go list -f '{{.Dir}}' -m | xargs -I{} bash -c 'cd {} && ${EXT_BIN_DIR}/golangci-lint run --config ${BASE_DIR}/.golangci.yaml'
```

Comment:
```
.PHONY: lint
lint:
	@echo -e "$(ATTN_COLOR)==> $@ $(NO_COLOR)"
	@${EXT_BIN_DIR}/golangci-lint config path
	@${EXT_BIN_DIR}/golangci-lint config verify
	@go work edit -json | jq -r '.Use[].DiskPath'  | xargs -I{} ${EXT_BIN_DIR}/golangci-lint run {}/... -c .golangci.yaml
```

Reply:
This causes:
```
WARN [runner] Can't run linter goanalysis_metalinter: musttag: running "go mod edit -json": exit status 1 
ERRO Running error: can't run linter goanalysis_metalinter
```
That is the reason I used a base dir as there is no go.mod in root

Reply:
You're getting that error because EXT_DIR needs to be defined as:
```
EXT_DIR            := ${PWD}/.ext
```

See here: https://github.com/aserto-dev/go-aserto/blob/main/makefile#L13